### ISSUE TITLE
Sometimes sorting of values is not wished

### ISSUE THREAD

**Issue by sebastiandeutsch** [DmitryBaranovskiy/g.raphael#39]
I propose an option field "dontSort" which can be set to true. The snippet I used for pie.js looks like this:

<pre>
if(!(opts['dontSort'] === true)) {
  values.sort(function (a, b) {
    return b.value - a.value;
  });
}
</pre>

I can create a patch if needed.

Regards

Sebastian

**Comment by ilikepi** [comment]
Agreed, it's not always desired to sort the values.  Sometimes it's preferable to display the values in a predictable order (i.e. the order in which they are specified).

**Comment by jboesch** [comment]
I agree. I have to implement a fix for this in my own code.  I did not want to sort it automatically cause it messed up my colors: [] array.  It would sort the labels but not the colors. Bah :(

**Comment by themasch** [comment]
I totally agree. Have the same problem with the colors. Someone may file a pull request?
